.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+package-lock.json
```

Comment:
I thought we wanted `package-lock.json` for library projects?

Reply:
Oh. I thought it was the other way around and that that we _didn't_ want it for library projects, only for applications 🤔

Reply:
Jon wrote this recently which I haven't yet fully parsed in my brain https://github.com/springernature/frontend-playbook/pull/368#issuecomment-696760068 but i _think_ he's saying that checking in package-lock.json for something like this would be undesirable (he wasn't able to offer me a link to find out more, sadly)

Reply:
Considering the potential to deprecate this repo if we ever manage to merge webservice and dashboard, it may be better to not add a package lock file at this stage, I think?